src/ansys/fluent/core/file_session.py

```diff
@@ -223,21 +223,23 @@ def get_surface_ids(self, surfaces: List[str | int]) -> List[int]:
             surfaces=surfaces,
         )
 
-    @deprecate_argument(
-        old_arg="surface_names",
-        new_arg="surfaces",
-        converter=lambda old_arg_val: old_arg_val,
-        warning_cls=PyFluentDeprecationWarning,
-    )
-    @deprecate_argument(
-        old_arg="surface_ids",
-        new_arg="surfaces",
-        converter=lambda old_arg_val: old_arg_val,
-        warning_cls=PyFluentDeprecationWarning,
-    )
-    @deprecate_arguments(
-        converter=_data_type_convertor,
-        warning_cls=PyFluentDeprecationWarning,
+    @all_deprecators(
```

Comment:
Somewhere we are parsing this information to generate the .rst sections for documentation. I prefer to keep the most logic in the parsing code and the source code logic should be simple and should avoid duplicate information. 
```
@deprecate_fn(
        new_fn="add_requests"
     )
@deprecate_argument(
        old_arg="surface_names",
        new_arg="surfaces",
        converter=lambda old_arg_val: old_arg_val,
        warning_cls=PyFluentDeprecationWarning,
    )
    @deprecate_argument(
        old_arg="surface_ids",
        new_arg="surfaces",
        converter=lambda old_arg_val: old_arg_val,
        warning_cls=PyFluentDeprecationWarning,
    )
```
We should be able to generate the deprecation message from the above decorator arguments.

Reply:
We are not parsing this information to generate the .rst sections for documentation for now, as per the discussion we are going to automate this in next task.